### ISSUE TITLE
Cheshire Cat pipeline allows obtaining all flags

### ISSUE THREAD

**Issue by daniel-beck** [cider-security-research/cicd-goat#16]
Running builds on the built-in node is a big security risk in Jenkins and strongly discouraged. As this environment has no protections for the built-in node, the Cheshire Cat pipeline's intended solution, perhaps inadvertently, gives the filesystem access needed to obtain all flags (with the exception of Duchess). This is perhaps not ideal (checklist item 4 in the readme points out this shouldn't be possible).

**Comment by asi-cider** [comment]
you're right and I was aware of this problem but I preferred back then to keep the challenge because of its importance. I've added another bullet to the instructions in the README file and a note in the challenge's description in CTFd. 
https://github.com/cider-security-research/cicd-goat/commit/f3b1e1ac12df60ad4d063749afd71cac464ec06d#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R70

If you'd like to suggest a better solution please share with us.

**Comment by daniel-beck** [comment]
I think the line added to the docs is a reasonable approach unless you're willing to consider separate installations for different flags. Thanks for that!
